diff --git a/nordnm/networkmanager.py b/nordnm/networkmanager.py
index eb847d9..181386b 100644
--- a/nordnm/networkmanager.py
+++ b/nordnm/networkmanager.py
@@ -233,16 +233,16 @@ def main():
 def set_killswitch(log=True):
     def main():
         killswitch_script = (
-            r'#!/bin/bash\n'
+            '#!/bin/bash\n'
             r'PERSISTENCE_FILE=' + paths.KILLSWITCH_DATA + '\n\n'
             r'case $2 in'
-            r'  vpn-up)\n'
-            r'    nmcli -f type,device connection | awk \'$1~/^vpn$/ && $2~/[^\-][^\-]/ { print $2; }\' > "${PERSISTENCE_FILE}"\n'
-            r'  ;;\n'
-            r'  vpn-down)\n'
-            r'    xargs -n 1 -a "${PERSISTENCE_FILE}" nmcli device disconnect\n'
-            r'  ;;\n'
-            r'esac\n')
+            '  vpn-up)\n'
+            r'    nmcli -f type,device connection | awk \'$1~/^vpn$/ && $2~/[^\-][^\-]/ { print $2; }\' > "${PERSISTENCE_FILE}"' + '\n'
+            '  ;;\n'
+            '  vpn-down)\n'
+            '    xargs -n 1 -a ' + r'"${PERSISTENCE_FILE}"' + ' nmcli device disconnect\n'
+            '  ;;\n'
+            'esac\n')
 
         try:
             with open(paths.KILLSWITCH_SCRIPT, "w") as killswitch:

From bc5faf9103eace3aeeebb45569ef26d881d7b894 Mon Sep 17 00:00:00 2001
From: evs-ch <evs-ch@users.noreply.github.com>
Date: Fri, 6 Mar 2020 15:50:34 +0100
Subject: [PATCH 3/3] bash / sh

---
 nordnm/networkmanager.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nordnm/networkmanager.py b/nordnm/networkmanager.py
index 181386b..1f98353 100644
--- a/nordnm/networkmanager.py
+++ b/nordnm/networkmanager.py
@@ -233,9 +233,9 @@ def main():
 def set_killswitch(log=True):
     def main():
         killswitch_script = (
-            '#!/bin/bash\n'
+            '#!/bin/sh\n'
             r'PERSISTENCE_FILE=' + paths.KILLSWITCH_DATA + '\n\n'
-            r'case $2 in'
+            'case $2 in\n'
             '  vpn-up)\n'
             r'    nmcli -f type,device connection | awk \'$1~/^vpn$/ && $2~/[^\-][^\-]/ { print $2; }\' > "${PERSISTENCE_FILE}"' + '\n'
             '  ;;\n'
